[rank: 0] Seed set to 0
Traceback (most recent call last):
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/utils/hub.py", line 478, in cached_files
    hf_hub_download(
    ~~~~~~~~~~~~~~~^
        path_or_repo_id,
        ^^^^^^^^^^^^^^^^
    ...<10 lines>...
        local_files_only=local_files_only,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py", line 1010, in hf_hub_download
    return _hf_hub_download_to_cache_dir(
        # Destination
    ...<14 lines>...
        force_download=force_download,
    )
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py", line 1117, in _hf_hub_download_to_cache_dir
    _raise_on_head_call_error(head_call_error, force_download, local_files_only)
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py", line 1649, in _raise_on_head_call_error
    raise LocalEntryNotFoundError(
    ...<2 lines>...
    )
huggingface_hub.errors.LocalEntryNotFoundError: Cannot find the requested files in the disk cache and outgoing traffic has been disabled. To enable hf.co look-ups and downloads online, set 'local_files_only' to False.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/main.py", line 183, in <module>
    cli_main()
    ~~~~~~~~^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/main.py", line 161, in cli_main
    LightningCLI(
    ~~~~~~~~~~~~^
        LightningModule,
        ^^^^^^^^^^^^^^^^
    ...<15 lines>...
        },
        ^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/main.py", line 111, in __init__
    super().__init__(*args, **kwargs)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/lightning/pytorch/cli.py", line 394, in __init__
    self.instantiate_classes()
    ~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/lightning/pytorch/cli.py", line 561, in instantiate_classes
    self.config_init = self.parser.instantiate_classes(self.config)
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_deprecated.py", line 140, in patched_instantiate_classes
    cfg = self._unpatched_instantiate_classes(cfg, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_core.py", line 1254, in instantiate_classes
    cfg[subcommand] = subparser.instantiate_classes(cfg[subcommand], instantiate_groups=instantiate_groups)
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_deprecated.py", line 140, in patched_instantiate_classes
    cfg = self._unpatched_instantiate_classes(cfg, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_core.py", line 1245, in instantiate_classes
    parent[key] = component.instantiate_classes(value)
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 625, in instantiate_classes
    value[num] = adapt_typehints(
                 ~~~~~~~~~~~~~~~^
        val,
        ^^^^
    ...<4 lines>...
        logger=self.logger,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 846, in adapt_typehints
    raise_union_unexpected_value(sorted_subtypes, val, vals)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/main.py", line 44, in _raise_union
    raise e
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 838, in adapt_typehints
    vals.append(adapt_typehints(val, subtype, **adapt_kwargs))
                ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1095, in adapt_typehints
    val = adapt_class_type(val, serialize, instantiate_classes, sub_add_kwargs, prev_val=prev_val)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1403, in adapt_class_type
    init_args = parser.instantiate_classes(init_args)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_deprecated.py", line 140, in patched_instantiate_classes
    cfg = self._unpatched_instantiate_classes(cfg, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_core.py", line 1245, in instantiate_classes
    parent[key] = component.instantiate_classes(value)
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 625, in instantiate_classes
    value[num] = adapt_typehints(
                 ~~~~~~~~~~~~~~~^
        val,
        ^^^^
    ...<4 lines>...
        logger=self.logger,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1095, in adapt_typehints
    val = adapt_class_type(val, serialize, instantiate_classes, sub_add_kwargs, prev_val=prev_val)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1403, in adapt_class_type
    init_args = parser.instantiate_classes(init_args)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_deprecated.py", line 140, in patched_instantiate_classes
    cfg = self._unpatched_instantiate_classes(cfg, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_core.py", line 1245, in instantiate_classes
    parent[key] = component.instantiate_classes(value)
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 625, in instantiate_classes
    value[num] = adapt_typehints(
                 ~~~~~~~~~~~~~~~^
        val,
        ^^^^
    ...<4 lines>...
        logger=self.logger,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1095, in adapt_typehints
    val = adapt_class_type(val, serialize, instantiate_classes, sub_add_kwargs, prev_val=prev_val)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_typehints.py", line 1417, in adapt_class_type
    return instantiator_fn(val_class, **{**init_args, **dict_kwargs})
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_common.py", line 224, in __call__
    return default_class_instantiator(class_type, *args, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/jsonargparse/_common.py", line 213, in default_class_instantiator
    return class_type(*args, **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/models/vit.py", line 27, in __init__
    AutoModel.from_pretrained(
    ~~~~~~~~~~~~~~~~~~~~~~~~~^
        backbone_name,
        ^^^^^^^^^^^^^^
    ),
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/models/auto/auto_factory.py", line 549, in from_pretrained
    config, kwargs = AutoConfig.from_pretrained(
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~^
        pretrained_model_name_or_path,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        **kwargs,
        ^^^^^^^^^
    )
    ^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/models/auto/configuration_auto.py", line 1288, in from_pretrained
    config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/configuration_utils.py", line 662, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
                          ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/configuration_utils.py", line 721, in _get_config_dict
    resolved_config_file = cached_file(
        pretrained_model_name_or_path,
    ...<10 lines>...
        _commit_hash=commit_hash,
    )
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/utils/hub.py", line 321, in cached_file
    file = cached_files(path_or_repo_id=path_or_repo_id, filenames=[filename], **kwargs)
  File "/lustrefs/disk/home/pjakkraw/workspace/eomt/.venv/lib/python3.13/site-packages/transformers/utils/hub.py", line 552, in cached_files
    raise OSError(
    ...<3 lines>...
    ) from e
OSError: We couldn't connect to 'https://huggingface.co' to load the files, and couldn't find them in the cached files.
Check your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
